doc/conf.py

```diff
@@ -33,6 +33,7 @@
 
 pygments_style = 'sphinx'
 
+html_file_suffix = ''
```

Comment:
Hmm -- on my Fedora 31 box, this change results in *all* the HTML files being generated w/o the `.html` suffix (and therefore no longer served from `admin/serve-docs`

Reply:
Yes, unfortunately I noticed this too yesterday on fedora and Bionic

Reply:
Fixed by #33562